kolibri/plugins/learn/assets/src/views/Breadcrumbs.vue

```diff
@@ -48,67 +58,25 @@
           { text: this.contentTitle },
         ];
       },
-      middleTopicBreadcrumbs() {
-        let crumbs = [];
-
-        // Channels have no previous topics
-        if (this.pageName === PageNames.TOPICS_CHANNEL) {
-          return crumbs;
-        }
-
-        // Link to top-level Channel
-        crumbs.push({
-          text: this.channelTitle,
-          link: {
-            name: PageNames.TOPICS_CHANNEL,
-            params: {
-              channel_id: this.channelRootId,
-            },
-          },
-        });
-
-        // Links to previous topics
-        if (this.pageName === PageNames.TOPICS_CONTENT) {
-          crumbs = [...crumbs, ...this.topicCrumbLinks(this.contentCrumbs)];
-        } else if (this.pageName === PageNames.TOPICS_TOPIC) {
-          crumbs = [...crumbs, ...this.topicCrumbLinks(this.topicCrumbs)];
-        }
-        return crumbs;
-      },
-      lastTopicBreadcrumb() {
-        if (this.pageName === PageNames.TOPICS_CHANNEL) {
-          return { text: this.channelTitle };
-        } else if (this.pageName === PageNames.TOPICS_CONTENT) {
-          return { text: this.contentTitle };
-        } else if (this.pageName === PageNames.TOPICS_TOPIC) {
-          return { text: this.topicTitle };
-        }
-
-        return {};
-      },
       topicsBreadcrumbs() {
         return [
           // All Channels Link
           {
-            text: this.coreString('channelsLabel'),
-            link: { name: PageNames.TOPICS_ROOT },
+            text: this.learnString('libraryLabel'),
+            link: { name: PageNames.LIBRARY },
```

Comment:
Breadcrumbs in the topics page don't need to start until the user is down at least one folder level. No reference to Library page is needed, just starting with the return to the highest topic (channel) folder. Which I only know because Peter already provided me this feedback 😄 
<img width="328" alt="Screen Shot 2021-11-01 at 3 38 55 PM" src="https://user-images.githubusercontent.com/17235236/139731186-1b32d949-5e19-4d38-93ff-e39dbe9248c6.png">

Reply:
Have updated to remove the Library from the breadcrumbs.